app/jobs/autoscaling.py

```diff
@@ -406,11 +411,12 @@ def remove_scalein_protection(instance_ids):
         return instance_ids
 
 if __name__ == "__main__":
-    assert len(sys.argv) > 2
-    assert sys.argv[1] in ("update", "debug")
-    assert sys.argv[2].isdigit()
-    assert sys.argv[4].isdigit()
-    assert sys.argv[6].isdigit()
-    if sys.argv[1] == "debug":
+    _, autoscaling_config_json = sys.argv
+    autoscaling_config = json.loads(autoscaling_config_json)
```

Comment:
This is actually pretty cool;   I didn't think you could put an entire json dict on the command line ;)